code/modules/clothing/spacesuits/rig/suits/mining.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/weapon/rig/mining
+	name = "intermediate hazard suit"
+	desc = "A stripped down, lighter version of the engineering suit. Commonly used during planet cracking operations or other \
+	 dangerous working environments to keep the wearer safe, while keeping their mobility."
+	icon_state = "ds_mining_rig"
+	armor = list(melee = 35, bullet = 50, laser = 50, energy = 15, bomb = 70, bio = 100, rad = 50)
```

Comment:
set melee,  bullet and laser armor to 7.5 points below the engineering suit, after the above changes